chef/data_bags/crowbar/migrate/nova/306_add_use_rbd_ephemeral.rb

```diff
@@ -0,0 +1,11 @@
+def upgrade(template_attributes, template_deployment, attributes, deployment)
+  key = "use_rbd_ephemeral"
+  attributes[key] = template_attributes[key] unless attributes.key? key
```

Comment:
```suggestion
  attributes[key] = false unless attributes.key? key
```
Then set the template to true, which would have the same effect as setting it in the model to true only on new installs. Not sure right now which one is better.

Reply:
Common pattern is to have migrations use default from template. I added enabling this for new deployments in `create_proposal()`